src/components/SearchBar/SearchList.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Fuse from 'fuse.js';
+import { useSelector } from 'react-redux';
+import { Resource } from '../../store/resourcesSlice';
+import PreviewCard from '../PreviewCard';
+import { RootState } from '../../store/reducers';
+
+
+function SearchList(props) {
+  const options = {
+    keys: ['title']
+  }
+  let resources = useSelector((state: RootState) => {
+    return Object.keys(state.resources).map<Resource>((r: string) => state.resources[r]);
+  });
+  if (props.video) {
+    resources = resources.filter((r) => r.type == 'Video');
+  }
+  if (props.article) {
+    resources = resources.filter((r) => r.type == 'Article');
+  }
+  const fuse = new Fuse(resources, options);
+  const results = fuse.search(props.query);
+  return (
```

Comment:
Can we add an empty state result that just renders "no results" or something similar if nothing comes up when you search?

Reply:
will do, fdkjghfdjkh DIDN'T KNOW WE WERE USING NPM AND NOT YARN MY BAD